ModalForm.php

```diff
@@ -30,6 +38,10 @@ class ModalForm extends Widget
 
     public $clientOptions;
 
+    private $_bsVersion;
+
+    public $forceBsVersion = false;
```

Comment:
This option may break on current installations. BS_4 was enabled by default.

Reply:
Hw can set bs4 [here](https://github.com/borodulin/yii2-modal-form/blob/c2a26fa5ea16bac77ff4bdbd0c9ea05f8e9be394/ModalFormAsset.php#L19) then ?

Reply:
I think yes

Reply:
bsVersion default is false and nothing change, only if u set true it load 
```
public $depends = [
        'yii\bootstrap4\BootstrapPluginAsset'
    ];
```

instead of
```
public $depends = [
        'yii\bootstrap\BootstrapPluginAsset'
    ];
```

It's a problem ?

Reply:
Ohh. I have not figured it out. I thought it was BS_4 by default.